src/Plugin/Patches.php

```diff
@@ -503,7 +503,13 @@ protected function executeCommand($cmd)
         $args = func_get_args();
         foreach ($args as $index => $arg) {
             if ($index !== 0) {
-                $args[$index] = escapeshellarg($arg);
+                if(is_array($arg)) {
```

Comment:
I tested this PR on FreeBSD and got an array to string conversion error, due to attempting to append a string to an array. Setting the arg to empty string so it can be appended to should resolve the error:

```suggestion
                if(is_array($arg)) {
                    $args[$index] = '';
```